PhotoService/ClientApp/src/components/profile-component/collections-component/Collections.js

```diff
@@ -4,27 +4,29 @@ import CollectionItem from './collection-item-component/CollectionItem';
 import profileService from '../../../services/profile.service';
 import './Collections.css'
 import AddCollectionDialog from './add-collection-dialog-component/AddCollectionDialog';
+import authService from '../../../services/auth.service';
 
 const Collections = ({username}) => {
     const [collections, setCollections] = useState([]);
     const [showDialog, setShowDialog] = useState(false);
     const navigate = useNavigate();
 
-    const handleCollectionClick = (id) => {
-        navigate(`./collections/view/${id}`);
+    const handleCollectionClick = (name) => {
+        navigate(`./collections/?collectionName=${name}`);
```

Comment:
better to use params
```suggestion
        navigate('./collections', { params: {collectionName: name}});
```